cookbooks/fb_postfix/attributes/default.rb

```diff
@@ -15,18 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-default['fb_postfix'] = {
-  'enable' => true,
-  'mask_service' => false,
-  'aliases' => {},
-  'access' => {},
-  'canonical' => {},
-  'etrn_access' => {
-    '127.0.0.1' => 'OK',
+main_cf = value_for_platform_family(
+  'debian' => {
+    'compatibility_level' => '3.6',
```

Comment:
Am I right in thinking we would need most of the `default` config as well

Reply:
I was able to send mail using the default `postfix` settings just fine!